src/videojs-media-sources.js

```diff
@@ -216,10 +236,12 @@ addTextTrackData = function (sourceHandler, captionArray, metadataArray) {
       // append muxed segments to their respective native buffers as
       // soon as they are available
       this.transmuxer_ = new Worker(videojs.MediaSource.webWorkerURI || '/src/transmuxer_worker.js');
+      this.transmuxer_.postMessage({action:'init', options: {remux: false}});
```

Comment:
This is *technically* the only thing that is required to get non-combined output!